packages/vuetify/src/components/VDataTable/VDataTableColumn.tsx

```diff
@@ -29,6 +30,7 @@ export const VDataTableColumn = defineFunctionalComponent({
           'v-data-table-column--last-fixed': props.lastFixed,
           'v-data-table-column--no-padding': props.noPadding,
           'v-data-table-column--nowrap': props.nowrap,
+          'd-none': props.hidden,
```

Comment:
This needs to be a new class, we don't use utility classes internally.

Reply:
Fixed!